core/src/main/java/bio/terra/pearl/core/dao/export/ExportOptionsDao.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.pearl.core.dao.export;
+
+import bio.terra.pearl.core.dao.BaseMutableJdbiDao;
+import bio.terra.pearl.core.model.export.ExportOptions;
+import org.jdbi.v3.core.Jdbi;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ExportOptionsDao extends BaseMutableJdbiDao<ExportOptions> {
+    public ExportOptionsDao(Jdbi jdbi) {
+        super(jdbi);
+    }
+
+    @Override
+    protected Class<ExportOptions> getClazz() {
+        return ExportOptions.class;
+    }
+
+    @Override
+    protected List<String> generateGetFields(Class<ExportOptions> clazz) {
+        List<String> getFields = super.generateGetFields(clazz);
+        getFields.add("excludeModules");
```

Comment:
Maybe this could be a separate field, `List<String> additionalColumns` or something, that the base jdbi dao picks up and adds so we don't have to override this method

Reply:
I think now that I know we can detect lists of strings, I'll just smarten up the logic so they're included by default

Reply:
(in a later PR)